ROMFS/px4fmu_common/init.d/CMakeLists.txt

```diff
@@ -51,6 +51,10 @@ px4_add_romfs_files(
 	rc.thermal_cal
 	rc.rover_apps
 	rc.rover_defaults
+	rc.rover_differential_apps
+	rc.rover_differential_defaults
+	rc.rover_ackermann_apps
```

Comment:
But then shouldnt we remove rover_apps and rover_defaults? Although I dont see enough difference that it should be split up if the only difference is just one module and all the rest is identical

Reply:
You are right, Silvan suggested the same thing. 

![image](https://github.com/PX4/PX4-Autopilot/assets/94360401/c3b00743-02d3-4249-8464-dd98afd036d4)

I don't have a strong opinion on if we should use just rover_apps or if we should split them up. If we were to use conditions in rover_apps and rover_defaults, how would they look like? 

Also there might not be much now, but if we want to "scale" and add more things later it might get crowded, but that might also be a problem for later if it even happens.

Reply:
> Also there might not be much now, but if we want to "scale" and add more things later it might get crowded, but that might also be a problem for later if it even happens.

I would prefer we split it if we actually have a problem. If the "later" doesn't happen as expected we just lose now with introducing too much structure without any advantange

Reply:
We removed the Ackermann split, thanks for the feedback :+1: